keras/backend/common.py

```diff
@@ -80,7 +80,8 @@ def set_floatx(floatx):
     """
     global _FLOATX
     if floatx not in {'float16', 'float32', 'float64'}:
-        raise ValueError('Unknown floatx type: ' + str(floatx))
+        raise ValueError('Unknown float datatype: ' + str(floatx))
+        raise ValueError('Only these data types are supported: float16, float32 and float64. Thanks! 😀')
```

Comment:
It is a good idea to list the supported types. A good error message should also print what was passed (like in the previous message). Also a good error message is informative and to the point, so no need for `Thanks! 😀` even though I do appreciate the positivity here :)